<?xml version="1.0" encoding="UTF-8"?>
<testrun name="CoolerTests" project="AutomaticRefiller" tests="13" started="13" failures="1" errors="0" ignored="0">
  <testsuite name="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.061">
    <testcase name="testRemoveBottlesFromCooler" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.009"/>
    <testcase name="testAddBottlesToCooler" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.006"/>
    <testcase name="testCountBottlesByType" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.001"/>
    <testcase name="testBottleManufacturing" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.002"/>
    <testcase name="testCoolerCapacity" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.001"/>
    <testcase name="testGetPercentFull" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.002"/>
    <testcase name="testSetGetBeveragePriceByType" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.002"/>
    <testcase name="testGetTotalStockValue" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.002"/>
    <testcase name="testGetStockValueByType" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.003"/>
    <testcase name="testSetDesiredMinimumStockForBeverage" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.005"/>
    <testcase name="testEditOrdersFromCooler" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.002"/>
    <testcase name="testSendOrdersFromCooler" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.002"/>
    <testcase name="testOverCapacityOrderVeto" classname="com.visionarysoftwaresolutions.refillable.CoolerTests" time="0.022">
      <failure>java.lang.AssertionError: Expected exception: java.lang.RuntimeException
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.runners.BlockJUnit4ClassRunner.runNotIgnored(BlockJUnit4ClassRunner.java:79)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:71)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</failure>
    </testcase>
  </testsuite>
</testrun>
